cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/bridge_to_ethereum_config.rs

```diff
@@ -65,6 +66,8 @@ parameter_types! {
 		rewards: Rewards { local: 1 * UNITS, remote: meth(1) },
 		multiplier: FixedU128::from_rational(1, 1),
 	};
+	pub GlobalAssetHub: Location = Location::new(1,[GlobalConsensus(RelayNetwork::get()),Parachain(rococo_runtime_constants::system_parachain::ASSET_HUB_ID)]);
```

Comment:
`GlobalAssetHub` is kind of confusing. Is this `AssetHubFromEthereum`?

Reply:
Agree, it make more sense.
https://github.com/paritytech/polkadot-sdk/pull/5546/commits/211b450f06fc8b29f44a50147f40e6544ccbeffa